be/src/vec/exec/jni_connector.cpp

```diff
@@ -311,8 +311,8 @@ Status JniConnector::_fill_column(TableMetaAddress& address, ColumnPtr& doris_co
     }
     MutableColumnPtr data_column;
     if (doris_column->is_nullable()) {
-        auto* nullable_column = reinterpret_cast<vectorized::ColumnNullable*>(
-                (*std::move(doris_column)).mutate().get());
```

Comment:
`mutate` will return the copied column when `use_count > 1`, so use `assume_mutable` to get the mutable column.